### PR TITLE
update package dependencies

### PR DESCRIPTION
**Pull request by shellscape**
## Rollup Plugin Name: many

This PR contains:

- [ ] bugfix
- [ ] feature
- [ ] refactor
- [ ] documentation
- [x] other

Are tests included?

- [ ] yes (_bugfixes and features will not be merged without tests_)
- [x] no

Breaking Changes?

- [ ] yes (_breaking changes will not be merged unless absolutely necessary_)
- [x] no

List any relevant issue numbers:

### Description

This is a repo-wide dependency update to get packages off of deprecated versions of plugins. 

**This PR should be straight merged and _not_ squash merged.** 

Individual publishes for packages that match `chore({name}): update dependencies` will follow after the merge. `pluginutils` was left out as it won't build with the new version of `commonjs` due to a regression (which I opened another issue for)

### CODE REVIEW

**Comment by shellscape** [comment]
I've no idea why CI is failing on `node-resolve` tests. Locally the `test`, `ci:test`, and `ci:coverage` scripts succeed. Please pull the branch and run `pnpm run ci:test` from repo root to verify.

**Comment by wuweiweiwu** [comment]
LGTM! Ci:tests pass locally for me as well.

**Comment by Kocal** [comment]
`pnpm ci:test` fails for me:
```
packages/node-resolve ci:test$ pnpm run test -- --verbose && pnpm run test:ts
│ Scope: current workspace package
│ > @rollup/plugin-node-resolve@7.0.0 pretest /home/hugo/workspace-os/plugins/packages/node-resolve
│ > pnpm run build
│ Scope: current workspace package
│ > @rollup/plugin-node-resolve@7.0.0 prebuild /home/hugo/workspace-os/plugins/packages/node-resolve
│ > del-cli dist
│ > @rollup/plugin-node-resolve@7.0.0 build /home/hugo/workspace-os/plugins/packages/node-resolve
│ > rollup -c
│ src/index.js → dist/index.js, dist/index.es.js...
│ created dist/index.js, dist/index.es.js in 759ms
│ > @rollup/plugin-node-resolve@7.0.0 test /home/hugo/workspace-os/plugins/packages/node-resolve
│ > ava "--verbose"
│ node-resolve: setting options.module is deprecated, please override options.mainFields instead
│ node-resolve: setting options.jsnext is deprecated, please override options.mainFields instead
│ node-resolve: setting options.main is deprecated, please override options.mainFields instead
│   ✔ jail › mark module outside the jail as external
│   ✔ jail › bundle module defined inside the jail
│ node-resolve: setting options.jsnext is deprecated, please override options.mainFields instead
│ node-resolve: setting options.main is deprecated, please override options.mainFields instead
│ node-resolve: setting options.module is deprecated, please override options.mainFields instead
│ node-resolve: setting options.main is deprecated, please override options.mainFields instead
│ node-resolve: setting options.module is deprecated, please override options.mainFields instead
│ node-resolve: setting options.jsnext is deprecated, please override options.mainFields instead
│ node-resolve: setting options.main is deprecated, please override options.mainFields instead
│ node-resolve: setting options.module is deprecated, please override options.mainFields instead
│ node-resolve: setting options.main is deprecated, please override options.mainFields instead
│   ✔ order › respects order if given module,jsnext:main,main (111ms)
│   ✔ order › respects order if given jsnext:main, main (103ms)
│   ✔ order › keeps the order of [browser, module, jsnext, main] with all enabled (110ms)
│   ✔ order › finds and uses a dual-distributed .js & .mjs module (115ms)
│   ✔ dedupe-custom › can deduplicate custom module directory (107ms)
│   ✔ prefer-builtins › true allows preferring a builtin to a local module of the same name
│   ✔ prefer-builtins › warning when preferring a builtin module, no explicit configuration
│   ✔ deprecated › should support enabling "jsnext" field resolution
│   ✔ prefer-builtins › false allows resolving a local module with the same name as a builtin module
│   ✔ deprecated › should support disabling "main" field resolution
│   ✔ deprecated › should support disabling "module" field resolution
│   ✔ deprecated › options.jsnext still works with correct priority
│   ✔ deprecated › finds a module with module field
│   ✔ prefer-builtins › warns when importing builtins
│   ✔ deprecated › options.module still works with correct priority
│   ✔ get-package-info › populates info for browser (110ms)
│   ✔ test › ignores IDs with null character
│   ✔ get-package-info › populates info for module (117ms)
│   ✔ get-package-info › populates info for main (127ms)
│ node-resolve: setting options.main is deprecated, please override options.mainFields instead
│   ✔ dedupe › dedupes deep imports by package name if dedupe is set (119ms)
│   ✔ dedupe › single module version is bundled if dedupe is set (129ms)
│   ✔ dedupe › single module version is bundled if dedupe is set as a function (124ms)
│   ✔ dedupe › dedupes scoped deep imports by package name if dedupe is set (126ms)
│   ✔ dedupe › multiple module versions are bundled if dedupe is not set (128ms)
│   ✔ only › specify the only packages to resolve (102ms)
│   ✔ only › regex
│   ✔ symlinks › resolves symlinked packages
│   ✔ symlinks › preserves symlinks if `preserveSymlinks` is true
│   ✔ root-dir › deduplicates modules from the given root directory
│   ✔ browser › disregards top-level browser field
│   ✔ browser › allows use of the top-level browser field
│   ✔ browser › respects local browser field
│   ✔ browser › allows use of object browser field, resolving implicit `main`
│   ✔ browser › supports `false` in browser field
│   ✔ browser › allows use of object browser field, resolving `main` (110ms)
│   ✔ browser › allows use of object browser field, resolving replaced builtins (110ms)
│   ✔ browser › options.browser = true still works (113ms)
│   ✔ browser › allows use of the object browser field (114ms)
│   ✔ browser › allows use of object browser field, resolving nested directories (114ms)
│   ✔ browser › disregards object browser field (122ms)
│   ✔ browser › allows use of object browser field, resolving to nested node_modules (121ms)
│   ✔ browser › pkg.browser with mapping to prevent bundle by specifying a value of false (131ms)
│   ✔ test › throws error if local id is not resolved (277ms)
│   ✔ test › finds a file inside a package directory (310ms)
│   ✔ test › supports non-standard extensions (313ms)
│   ✔ test › allows custom options (313ms)
│   ✔ test › generates manual chunks (319ms)
│   ✔ test › resolves dynamic imports (321ms)
│   ✔ test › loads local directories by finding index.js within them (326ms)
│   ✔ test › finds a module with jsnext:main (336ms)
│   ✔ test › supports ./ in entry filename (326ms)
│   ✔ test › finds and uses an .mjs module (330ms)
│   ✔ test › finds and converts a basic CommonJS module (349ms)
│   ✔ test › handles a trailing slash (357ms)
│   ✖ test › handles package side-effects Did not match snapshot
│   ✔ test › ignores deep-import non-modules (389ms)
│   ✔ test › loads package directories by finding index.js within them (402ms)
│   1 test failed
│   test › handles package side-effects
│   /home/hugo/workspace-os/plugins/packages/node-resolve/test/fixtures/test/test.js:219
│   Did not match snapshot
│   Difference:
│     [
│   -   'false-dep1',
│   +   'array-dep1',
│   -   'true-dep1',
│   +   'array-dep3',
│   -   'true-dep2',
│   +   'array-dep5',
│   -   'true-index',
│   +   'array-index',
│   +   'false-dep1',
│   +   'true-dep1',
│   +   'true-dep2',
│   +   'true-index',
│     ]
│  ERROR  Test failed. See above for more details.
└─ Failed in 5s
/home/hugo/workspace-os/plugins/packages/node-resolve:
 ERROR  @rollup/plugin-node-resolve@7.0.0 ci:test: `pnpm run test -- --verbose && pnpm run test:ts`
Exit status 1
Error: Command failed with exit code 1 (EPERM): pnpm run ci:test --filter ./packages
    at makeError (/home/hugo/workspace-os/plugins/node_modules/.pnpm/registry.npmjs.org/execa/2.1.0/node_modules/execa/lib/error.js:59:11)
    at handlePromise (/home/hugo/workspace-os/plugins/node_modules/.pnpm/registry.npmjs.org/execa/2.1.0/node_modules/execa/index.js:112:26)
    at processTicksAndRejections (internal/process/task_queues.js:94:5)
    at async /home/hugo/workspace-os/plugins/scripts/run-changed.js:70:17 {
  command: 'pnpm run ci:test --filter ./packages',
  exitCode: 1,
  exitCodeName: 'EPERM',
  stdout: undefined,
  stderr: undefined,
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false,
  signal: undefined
}
Scope: current workspace package
```

I'm running node 12.14.1 and pnpm 4.7.1 on Debian 9.11

**Comment by shellscape** [comment]
OK thanks for checking. Since we've got a failure verified, I'm going to revert updates to that one and try to tackle it in a separate PR.

**Comment by shellscape** [comment]
@wuweiweiwu @Kocal I removed the `node-resolve` commit and will handle that separately in another branch/PR. Please have another look and let me know if you're good with merge.

File: packages/inject/package.json [hunk 1]
```diff
@@ -46,17 +46,17 @@
     "rollup": "^1.20.0"
   },
   "dependencies": {
-    "estree-walker": "^1.0.1",
-    "magic-string": "^0.25.2",
-    "rollup-pluginutils": "^2.6.0"
+    "@rollup/pluginutils": "^3.0.4",
+    "estree-walker": "1.0.1",
```
Comment (wuweiweiwu) [inline packages/inject/package.json]: is this meant to be a version lock on `1.0.1`?

File: packages/auto-install/README.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ export default {
 };
 ```
 
-_Note: ensure that this plugin is added to the `plugins` array *before* [rollup-plugin-node-resolve](https://github.com/rollup/rollup-plugin-node-resolve)._
+_Note: ensure that this plugin is added to the `plugins` array *before* [@rollup/plugin-node-resolve](https://github.com/rollup/plugins)._
```
Comment (NotWoods) [inline packages/auto-install/README.md]: nit: Link to the homepage for node-resolve
```suggestion
_Note: ensure that this plugin is added to the `plugins` array *before* [@rollup/plugin-node-resolve](https://github.com/rollup/plugins/tree/master/packages/node-resolve)._
```

**Pull request merged**